criu/uffd.c

```diff
@@ -1485,6 +1485,5 @@ int cr_lazy_pages(bool daemon)
 
 	tls_terminate_session();
 
-	xfree(events);
```

Comment:
I think it would be good to have a comment to explain why we don't free events after handle_requests.

nit: we can move this xfree in handle_requests.